src/ScriptingCore/utf8_tools.cpp

```diff
@@ -17,24 +17,39 @@ Copyright 2009 Dan Weatherford, Facebook inc
 
 #include <limits.h>
 #include <boost/algorithm/string/case_conv.hpp>
-#include <locale>
-#include <codecvt>
+#include <boost/locale/encoding_utf.hpp>
+#include <string>
+
+///#include <codecvt>
+//#include <locale>
 
 #include "utf8_tools.h"
 
-std::wstring_convert<std::codecvt_utf8<wchar_t>> utf8_conv;
```

Comment:
The version here is using built-in C++11 routines to do the conversion; I'd rather not rely on the boost conversions unless there is a particularly strong reason to do so.

What is the reason you've changed this?